sdk/python/feast/feature_view.py

```diff
@@ -521,6 +524,11 @@ def from_proto(cls, feature_view_proto: FeatureViewProto):
                 )
             )
 
+        if feature_view_proto.spec.entity_key_serialization_version <= 1:
+            feature_view.set_entity_key_serialization_version(1)
```

Comment:
Probably add some type of explanation here to differentiate what these versions mean.

Reply:
Added docs in the repo_config